### ISSUE TITLE
The doc of `torch.nn.RReLU` does not demonstrate its difference between training and eval

### ISSUE THREAD

**Issue by SiriusNEO** [pytorch/pytorch#95605]
### 📚 The doc issue

I'm reading the doc of [Randomized Leaky ReLU](https://pytorch.org/docs/stable/generated/torch.nn.RReLU.html#torch.nn.RReLU). As we know, its behavior is different in training mode and eval mode. In training mode `alpha` is sampled from an uniform distribution while in evaluation mode `alpha = (lower + upper) / 2`. I think the documentation can add some illustrations about this.

### Suggest a potential alternative/fix

The original doc in RReLU:
```
where :math:`a` is randomly sampled from uniform distribution
    :math:`\mathcal{U}(\text{lower}, \text{upper})`.
```
Can be modified as:
```
where :math:`a` is randomly sampled from uniform distribution
    :math:`\mathcal{U}(\text{lower}, \text{upper})` during training while during 
evaluation :math:`a` is fixed with :math:`a = \frac{\text{lower} + \text{upper}}{2}`.
```

cc @svekars @carljparker

**Comment by ezyang** [comment]
Sure, send us a PR
